app/assets/javascripts/analytics/error-tracking.js

```diff
@@ -0,0 +1,24 @@
+// Extension to track errors using google analytics as a data store.
+(function() {
+
+    "use strict";
+    var trackJavaScriptError = function (e) {
+      var errorSource = e.filename + ': ' + e.lineno;
+      _gaq.push([
+        '_trackEvent',
+        'JavaScript Error',
```

Comment:
Cool, I'm the one responsible for 'Print Intent' so I guess this is my second motion for Title Cased Names